src/util/config/build.js

```diff
@@ -25,6 +25,21 @@ export const schema = {
 			arg: 'asset-port',
 			env: process.env.NODE_ENV !== 'test' && 'ASSET_SERVER_PORT', // don't read env in tests
 		},
+		protocol: {
```

Comment:
This is just adding some defaults to the schema. This prevents the build from complaining about these values not being defined when it starts the server.